chapter_optimization/lr-scheduler.md

```diff
@@ -427,8 +430,8 @@ scheduler = CosineScheduler(max_update=20, base_lr=0.3, final_lr=0.01)
 d2l.plot(d2l.arange(num_epochs), [scheduler(t) for t in range(num_epochs)])
 ```
 
-在计算机视觉中，这个调度可以引出改进的结果。
-但请注意，如下所示，这种改进并不能保证成立。
+在计算机视觉的背景下，这个调度方式可能产生改进的结果。
+但请注意，如下所示，并不能保证这种改进一定成立。
```

Comment:
```suggestion
但请注意，如下所示，这种改进并不一定成立。
```

Reply:
Have finished commit merging and been built yet.